pipeline/load.py

```diff
@@ -50,12 +50,8 @@ def load_price_changes(products_data: list[dict], conn: connection) -> None:
         logging.error("Load error during commit: %s", commit_error)
 
 
-def main_load():
+def main_load(cleaned_data: list[dict]) -> None:
```

Comment:
Great that you've moved away from hard coding in test data and allow for inputs!